### ISSUE TITLE
(partition Number RealNumber ImaginaryNumber ComplexNumber)

### ISSUE THREAD

**Issue by jefft0** [ontologyportal/sumo#39]
Merge.kif says:

```
(partition Number RealNumber ImaginaryNumber ComplexNumber)
```

https://github.com/ontologyportal/sumo/blob/master/Merge.kif#L1542

I don't know how you express a complex number in SUMO but suppose "0,1 + 0.2i" is (ComplexFn 0.1 0.2). If the imaginary component is zero, do you still consider it to be a complex number disjoint from a real number? In other words, does the following hold?

```
(equals 0.1 (ComplexFn 0.1 0))
```

If so, then the partition above is wrong. One might want the simpler partition:

```
(partition Number RealNumber ImaginaryNumber)
```

I don't know how you make an imaginary number but suppose we have ImaginaryFn. Again, does the following hold?

```
(equals 0.0 (ImaginaryFn 0.0))
```

In other words, is zero equal to 0i? If so then the above partition of RealNumber and ImaginaryNumber is also wrong.

I do think the following is true, if you think that numbers along the real axis are a subset of numbers in the complex plane:

```
(subclass RealNumber ComplexNumber)
(subclass ImaginaryNumber ComplexNumber)
```

**Comment by apease** [comment]
we have ImaginaryPartFn and RealNumberFn to handle expressing the two parts of a complex number.  But developing the axiomatic support for them and a set of useful examples would be a very welcome PR!  Closing this issue for now
